data/strings/types_strings.txt

```diff
@@ -4174,7 +4174,7 @@
     fi = Kierrätysjäteastia
     fr = Conteneur de recyclage
     hi = रीसाइक्लिंग कंटेनर
-    hu = Újrahasznosító tartály
+    hu = Újrahasznosító edény
```

Comment:
Not "edény" or "tartály", instead use "konténer".

> Valahányszor többet látnak újrahasznosító **_konténerek_** az utcán, mivel az emberek fokozatosan tudatosulnak és elkezdik újrahasznosítani, bár a legújabbaknál mindig ugyanazok a kétségek vannak.
_From: https://www.renovablesverdes.com/hu/%C3%BAjrahasznos%C3%ADt%C3%B3-kuk%C3%A1k/_